capture/engine_scripts/puppet/clickAndHoverHelper.js

```diff
@@ -7,31 +7,31 @@ module.exports = async (page, scenario) => {
 
   if (keyPressSelector) {
     for (const keyPressSelectorItem of [].concat(keyPressSelector)) {
-      await page.waitFor(keyPressSelectorItem.selector);
+      await page.waitForTimeout(keyPressSelectorItem.selector);
```

Comment:
Should `page.waitForSelector` be used here?

Reply:
That indeed not backward compatible, if someone is using an xpath expression rather then a CSS selector, this previously worked. But would no longer work afterwards.